test/custom-naming-test.js

```diff
@@ -20,12 +20,13 @@ describe('Custom delimeters for BEM naming', function() {
         block: 'b1',
         mix: { elem: 'e2' }
       }
-    ], '<div class="b1$$e1 b1$$e1@@a@@b"></div>' +
+    ], '<div class="b1$$e1 b1$$e1@@a**b"></div>' +
       '<div class="b1$$e1"></div>' +
       '<div class="b1 b1$$e2"></div>', {
       naming: {
         elem: '$$',
-        mod: '@@'
+        mod: '@@',
+        modVal: '**'
```

Comment:
modVal is a new standard.

Existing variants:
```
{ mod, val }
{ mod: { name, val } }
```

I'd stick to https://github.com/bem-sdk-archive/bem-naming#delims — `{mod: string}` or `{mod: {name, val}}`